<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>339</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>323</width>
    <height>339</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>323</width>
    <height>339</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Лабораторная 4</string>
  </property>
  <widget class="QTableWidget" name="qTableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>302</width>
     <height>122</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>302</width>
     <height>122</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>302</width>
     <height>122</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>302</width>
     <height>122</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <stylestrategy>NoAntialias</stylestrategy>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>4</number>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>60</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>60</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="qPushButtonRandom">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Случайные
числа</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qPushButtonSolver">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Выполнить
задание</string>
   </property>
  </widget>
  <widget class="QLabel" name="qLabelUslovie">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>NoAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Найти максимальный элемент второй строки.&lt;br&gt;Если он больше первого элемента третьей строки,&lt;br&gt;то поменять элементы местами</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="qLabelMaxItemValue">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Максимальный&lt;br&gt;элемент 2-й строки</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
